### ISSUE TITLE
`Error Something went wrong` when uploading mp4 and pdf files

### ISSUE THREAD

**Issue by ElliotBadinger** [n4ze3m/dialoqbase#188]
Getting the error in title when I try to upload bite-sized mp4 files (about 7 MB) and pdfs 
Still encounter the issue when I try to upload each file one by one.

**Comment by n4ze3m** [comment]
Hey, There is an issue caused by Swagger integration, I guess it will be fixed in tomorrow's update. Sorry about that

**Comment by ElliotBadinger** [comment]
Thanks. Great work on your project by the way :)

A quick tangent: How did you ascertain the programming knowledge you have to build this project?

**Comment by n4ze3m** [comment]
thanks haha

> A quick tangent: How did you ascertain the programming knowledge you have to build this project?

I'm a self-taught programmer on a continuous journey to enhance my skills by actively engaging in the creation of diverse projects.  Building projects allows me to tackle challenges, experiment with different solutions, and ultimately refine my programming expertise.

**Comment by n4ze3m** [comment]
Hey, are you using the latest version, v1.6.0?

**Comment by ElliotBadinger** [comment]
Yep

**Comment by n4ze3m** [comment]
Okk, You are using Docker, right?

**Comment by ElliotBadinger** [comment]
Yes I am

**Comment by ElliotBadinger** [comment]
Maybe this could be of assistance (my API keys for all models -- with the exception of OpenAI as I did not set its key --were set successfully)
![image](https://github.com/n4ze3m/dialoqbase/assets/114085572/a622d08c-3bf3-4468-b4cf-8bba6ca8d2ca)

**Comment by phpia** [comment]
Hi! Getting some errors too uploading files. 
![Captura desde 2024-01-15 18-12-22](https://github.com/n4ze3m/dialoqbase/assets/43242385/801d8169-0f28-4d42-9574-0a325b0b118c)

**Comment by n4ze3m** [comment]
Hey, is the version causing the issue 1.6.1, right

**Comment by ElliotBadinger** [comment]
Yes, as well as the previous version

**Comment by phpia** [comment]
Yes

**Comment by n4ze3m** [comment]
Hey, The issue has been resolved in version 1.7.0. Please let me know if it still occurs

**Comment by ElliotBadinger** [comment]
Updated to the latest version using `docker-compose pull` (after stopping the program) and I still get the error regardless.

**Comment by n4ze3m** [comment]
Hey, Is the application showing version 1.7.0 correctly, and is the error coming from the 'Add New Datasource' page?

**Comment by ElliotBadinger** [comment]
Yes it is.![Screenshot_20240120-150253~2.png](https://github.com/n4ze3m/dialoqbase/assets/114085572/ed53c785-7174-49b8-97fe-90a95317b853)

And yes to the second question

**Comment by ElliotBadinger** [comment]
The error pops up even when I try to add files whilst in the process of creating a new bot
